_posts/2020-4-17-pytorch-1-dot-5-released-with-new-and-updated-apis.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog_detail
+title: 'PyTorch 1.5 released, new and updated APIs including C++ frontend API parity with Python'
+author: Team PyTorch
+---
+
+
+Today, we’re announcing the availability of PyTorch 1.5, along with new and updated libraries. This release includes several major new API additions and improvements. PyTorch now includes a significant update to the C++ frontend, ‘channels last’ memory format for computer vision models, and a stable release of the distributed RPC framework used for model-parallel training. The release also has new APIs for autograd for hessians and jacobians, and an API that allows the creation of Custom C++ Classes that was inspired by PyBind.
```

Comment:
nit: I think it's spelled [pybind](https://pybind11.readthedocs.io/en/stable/intro.html) instead of PyBind.